app/src/main/res/layout-land/hints_summary.xml

```diff
@@ -34,7 +34,6 @@
         style="@style/TextViewStart"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_gravity="center"
```

Comment:
Won't this create issue as the style is `TextViewStart` and we are removing the gravity which is center?